pkgs/servers/trezord/default.nix

```diff
@@ -13,14 +13,14 @@ buildGoPackage rec {
     owner  = "trezor";
     repo   = "trezord-go";
     rev    = "v${version}";
-    sha256 = "03pz223jjfbd0g9gkk21q6d27jc8vd1bc1jz00i0f3dzvsyfx5g6";
+    sha256 = "19am5zs2mx36w2f8b5001i1sg6v72y1nq5cagnw6rza8qxyw83qs";
   };
 
   meta = with stdenv.lib; {
     description = "TREZOR Communication Daemon aka TREZOR Bridge";
-    homepage = https://mytrezor.com;
-    license = licenses.gpl3;
-    maintainers = with stdenv.lib.maintainers; [ canndrew jb55 ];
+    homepage = https://trezor.io;
+    license = licenses.lgpl3;
+    maintainers = with maintainers; [ canndrew jb55 "1000101"];
```

Comment:
Resistance is futile, I don't think you can avoid writing `maintainers."1000101"``

Reply:
Any idea on how to do it properly? Apart from changing my github name.

Reply:
```
maintainers = with maintainers; [ canndrew jb55 maintainers."1000101"];
```

Reply:
Technically speaking, your `maintainers` entry can be different from your github username (I am an example)

Reply:
Oh, you're absolutely right. But let's have some fun :).